### ISSUE TITLE
Error with tweak mode when using underscores in numbers

### ISSUE THREAD

**Issue by TFC-343** [benfry/processing4#442]
## Description
An error when running tweak mode.

## Expected Behavior
I expect code that runs in normal mode, to not raise a syntax error in tweak mode.

## Current Behavior
When running tweak mode, under a specific set of circumstances, it can raise an error when running normal does not.
It seems that when using underscore syntax for numbers, `1_000` tweak mode does not probably convert this to a tweak integer. 

## Steps to Reproduce
I have found this code to consistently throw the error
```
void setup(){
  size(100, 100);
}

void draw(){
 int x = 1_000;

}
```
the error raised is `Syntax Error - Missing name or ; near ‘ int x = tweakmode_int[0]_000’?`
This error does not happen if there is no `size()` call, other calls or assignments in setup do not raise the error. 
The code does not need to assign 1_000, it can try and pass it to a function or output it.
Underscores in other places in the number also raise the error.

## Your Environment

 

* Processing version:  4.0b7
* Operating System and OS version: Windows 11 Home, 21H2

**Comment by arnoudvanderleer** [comment]
I will take a look at this

**Comment by benfry** [comment]
Fix from @arnoudvanderleer merged for 4.0 beta 8.
